util/dvsim/Modes.py

```diff
@@ -509,7 +509,7 @@ def create_regressions(regdicts, sim_cfg, tests):
                 # which case, skip
                 if run_mode_obj.name in sim_cfg.en_run_modes:
                     continue
-                self.run_opts.extend(run_mode_obj.run_opts)
+                regression_obj.run_opts.extend(run_mode_obj.run_opts)
```

Comment:
Ok, done (issue #2056). I don't know the steps to reproduce the bug, I'm afraid. I *think* this is the right fix, but to be certain, either the author of the code (you?) or a user (who can't yet exist, because it doesn't work) would need to check it.